build/travis/job_macos_lupdate/script.sh

```diff
@@ -33,5 +33,4 @@ rm tours.pro
 # username = $TRANSIFEX_USER
 # EOL
 
-# cp share/locale/mscore_en_US.ts share/locale/mscore_en.ts
```

Comment:
This is the culprit for feeding back strings 'translated' into en_US as source strings for all languages, but then not having a matching real source, so the translations don't work at all for those strings in any translation